napari_aicssegmentation/core/router.py

```diff
@@ -19,9 +19,9 @@ def __init__(self, application: IApplication):
         self._workflow_engine = WorkflowEngine()
 
     def workflow_selection(self):
-        self._controller = WorkflowSelectController(self._application, self._layer_reader, self._workflow_engine)
+        self._controller = WorkflowSelectController(self._application, LayerReader(), WorkflowEngine())
         self._controller.index()
 
-    def workflow_steps(self):
-        self._controller = WorkflowStepsController(self._application, self._workflow_engine)
+    def workflow_steps(self, workflow: str):
```

Comment:
Same as previous comments - this is not necessary since we will read the workflow from the model.